curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641daa8c2c3e364ac3650b37.md

```diff
@@ -17,11 +17,6 @@ Add a `div` element above the author's bio and give it the `class` `"purple-divi
 window.fetch = () => Promise.resolve({json: () => Promise.resolve([{ author: 'Whoever', image: 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///ywAAAAAAQABAAACAUwAOw==', url: "http://not-a-real-url.nowhere/", bio: 'words go here' }])});
 ```
 
-# --before-all--
-
-```js
-window.fetch = () => Promise.resolve({json: () => Promise.resolve([{ author: 'Whoever', image: 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///ywAAAAAAQABAAACAUwAOw==', url: "http://not-a-real-url.nowhere/", bio: 'words go here' }])});
-```
 
```

Comment:
```suggestion
```
For consistency please leave only single blank line between end of code block (` ``` `) and start of the next section (`# --hints--`). This applies to other modified files as well.